srcpkgs/git-lfs/template

```diff
@@ -1,52 +1,24 @@
 # Template file for 'git-lfs'
 pkgname=git-lfs
-version=2.12.1
-revision=2
+version=3.3.0
+revision=1
 build_style=go
-go_import_path="github.com/git-lfs/git-lfs"
-hostmakedepends="git ruby-ronn"
+go_import_path="github.com/git-lfs/git-lfs/v3"
+hostmakedepends="git ruby-asciidoctor"
 short_desc="Git extension for versioning large files"
 maintainer="Michael Aldridge <maldridge@voidlinux.org>"
 license="MIT"
 homepage="https://git-lfs.github.com/"
 distfiles="https://github.com/git-lfs/git-lfs/archive/v${version}.tar.gz"
-checksum=2b2e70f1233f7efe9a010771510391a07527ec7c0af721ecf8edabac5d60f62b
+checksum=d5eeb9ee33188d3dd6a391f8a39b96d271f10295129789e5b3a1ac0e9f5114f5
 
 post_build() {
 	make man
```

Comment:
https://github.com/void-linux/void-packages/actions/runs/3985990165/jobs/6834016287#step:7:2656
```
fatal: not a git repository (or any of the parent directories): .git
...
```
Can be fixed by using
```
VERSION="${version}" make man
```

Reply:
See https://github.com/git-lfs/git-lfs/blob/02ac3de0e19fbfeea7f8ddb079cfe074367a48d3/Makefile#L8

Reply:
good catch - the error was already present in v2 but seemed not to be fatal.

done.